### PR TITLE
statsd: remove deprecated user/project ids opts

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

File: gnocchi/opts.py [hunk 1]
```diff
@@ -176,16 +176,7 @@ def list_opts():
...
-                'user_id',
-                deprecated_for_removal=True,
-                help='User ID to use to identify statsd in Gnocchi'),
-            cfg.StrOpt(
-                'project_id',
-                deprecated_for_removal=True,
-                help='Project ID to use to identify statsd in Gnocchi'),
-            cfg.StrOpt(
                 'creator',
-                default="${statsd.user_id}:${statsd.project_id}",
```
Comment (chungg) [inline gnocchi/opts.py]: where are these values coming from?

Comment (jd) [inline gnocchi/opts.py]: the config file. it's `[statsd]user_id`. oslo.config supports variable substitution

Comment (chungg) [inline gnocchi/opts.py]: does the default become ':' if they're not there then?

Comment (jd) [inline gnocchi/opts.py]: Yes.
(though you realize your commenting the *current* code, not my change right? :))

Comment (chungg) [inline gnocchi/opts.py]: oh. word. i saw green for some reason.lol

**Pull request merged**